checks/microbenchmarks/cpu/simd/nsimd.py

```diff
@@ -60,6 +58,10 @@ class NsimdTest(rfm.RegressionTest):
     def set_descr(self):
         self.descr = f'testing {self.bench_name}'
 
+    @run_before('compile')
+    def skip_test(self):
+        self.skip_if(True, 'skipping legacy test (nsimd no longer installed)')
```

Comment:
Not really: we stopped installing it -> there is no modulefile. 
I want to keep the message so that we remember but we could also get rid of that test, you decide.